resources/todomvc/architecture-examples/preact/rollup.config.js

```diff
@@ -1,27 +1,53 @@
-import nodeResolve from 'rollup-plugin-node-resolve';
-import commonjs from 'rollup-plugin-commonjs';
-import babel from 'rollup-plugin-babel';
+import terser from "@rollup/plugin-terser";
+import resolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import babel from "@rollup/plugin-babel";
+import css from "rollup-plugin-import-css";
+import copy from "rollup-plugin-copy";
+
+// `npm run build` -> `production` is true
+// `npm run dev` -> `production` is false
+const production = !process.env.ROLLUP_WATCH;
+console.log(production, process.env.ROLLUP_WATCH);
```

Comment:
If we want to keep this log, what do you think of making it somewhat prettier with some prefix?
Or remove it :-)

Reply:
ha - let's remove - thanks for catching this 😄